pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -237,6 +237,8 @@ func manageServiceAccountCABundle(lister corev1listers.ConfigMapLister, client c
 		// include the ca bundle needed to recognize default
 		// certificates generated by cluster-ingress-operator
 		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.GlobalMachineSpecifiedConfigNamespace, Name: "router-ca"},
+		// include ca bundle from service-ca
+		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.GlobalMachineSpecifiedConfigNamespace, Name: "service-ca"},
```

Comment:
The `ca.crt` should not contain the `service-ca` because the trust domains are different.  The service-ca.crt is a separate file because it is used for a different purpose and signs indiscriminately.  

This should match 3.x and place the service-ca into `service-ca.crt`.

Reply:
Ok, that makes sense. Where is the right spot to wire that in?